src/Scorecard.js

```diff
@@ -0,0 +1,43 @@
+function strikeOrSpare(frame){
+  return (frame[0] + frame[1] == 10 ? true : false);
+}
+function strike(frame){
+  return (frame[0] == 10 ? true : false );
+}
+// takes in a nested arr.
+function Scorecard(bowls){
```

Comment:
thats a very big if/else statment. do you think you could refactor this in smaller funcions?